microsoft/testsuites/performance/kvmperf.py

```diff
@@ -210,229 +209,176 @@ def perf_nested_kvm_ntttcp_different_l1_nat(
         ) = parse_nested_image_variables(variables)
 
         try:
-
-            # setup nat on client and server
-            self._setup_nat(
-                server_l1,
-                self._BR_NAME,
-                self._BR_NETWORK,
-                self._BR_CIDR,
-                self._BR_ADDR,
-                f"{self._SERVER_IP_ADDR},{self._SERVER_IP_ADDR}",
-                self._SERVER_TAP,
-            )
-            self._setup_nat(
-                client_l1,
-                self._BR_NAME,
-                self._BR_NETWORK,
-                self._BR_CIDR,
-                self._BR_ADDR,
-                f"{self._CLIENT_IP_ADDR},{self._CLIENT_IP_ADDR}",
-                self._CLIENT_TAP,
-            )
-
-            # setup l2 vm on server and client
-            server_l2 = connect_nested_vm(
+            # setup nested vm on server in NAT configuration
+            server_l2 = self._setup_nat(
                 server_l1,
+                "server_l2",
                 nested_image_username,
                 nested_image_password,
                 self._SERVER_HOST_FWD_PORT,
                 nested_image_url,
-                image_name=self._SERVER_IMAGE,
-                taps=[self._SERVER_TAP],
-                stop_existing_vm=True,
-                name="server",
+                self._SERVER_IP_ADDR,
+                self._BR_NAME,
+                self._BR_NETWORK,
```

Comment:
When there are many arguments, it's better to specify names of them. It's easier to read.